### ISSUE TITLE
Unspecified error

### ISSUE THREAD

**Issue by feinfinger** [tango-controls/waltz#171]
Sometimes unspecified errors pop up although the command is issued correctly/ This behaviour is seen when trying to move a motor.
![screenshot from 2018-12-19 09-59-32](https://user-images.githubusercontent.com/468785/50209979-51351100-0375-11e9-8c9a-4ff415326913.png)

**Comment by Ingvord** [comment]
@feinfinger , thank you very much for reporting this issue!

Further info:

There is an internal server error:

```
[19/Dec/2018:09:57:27 +0100] "PUT /tango/rest/rc4/hosts/hzgpp05vme2/10000/devices/p05/tripod/pods/attributes/value?PosZ=30&async=true HTTP/1.1" 500 2589
```

Unfortunately, due to [rest-server#178](https://github.com/tango-controls/rest-server/issues/178) there is a total mess with logs (there is no logs :( ) No more information can be provided at this time

**Comment by Ingvord** [comment]
Should be closed via [tango-controls/rest-server commit 07d8c26f](https://github.com/tango-controls/rest-server/commit/07d8c26f1567c61851093694d1efd879e2e109dd)
